python-usfm-parser/src/usfm_grammar/usfm_parser.py

```diff
@@ -57,7 +57,8 @@ class Format(str, Enum):
 TABLE_CELL_MARKERS = ["tc", "th", "tcr", "thr"]
 
 ANY_VALID_MARKER = PARA_STYLE_MARKERS+NOTE_MARKERS+CHAR_STYLE_MARKERS+\
-                    NESTED_CHAR_STYLE_MARKERS+TABLE_CELL_MARKERS
+                    NESTED_CHAR_STYLE_MARKERS+TABLE_CELL_MARKERS+\
+                    ["fig", "cat", "esb", "b", "ph", "pi"]
```

Comment:
Any reason why these markers are not categorized like the rest?

Reply:
First 4 could be called special cases maybe. Pi and pH are para markers but with intendation. So in short "special" would be the only term to call them collectively. Nothing much in common.

Reply:
Okay, so maybe calling them `MISC_MARKERS` (for miscellaneous) or something such?

Reply:
Changed :+1: